### DISCUSSION TITLE
Can I proxy the 25 port with nginx?

### DISCUSSION THREAD

**Discussion by openaspace** [modoboa/modoboa#2649] [Q&A]
Hi.
I run the mail server with access to 993 and 587 only by VPN access with local IP and self signed SSL.

The only wan exposed port is the 25.

Can I change the 25 port to a higher port and run it with less user permissions and proxy the 25 port using nginx?

Thanx

**Comment by Spitfireap** [comment]
Hi, 

Replacing first `smtp` to the port you want in `master.cf` should do the trick.

**Comment by tonioo** [comment]
As far as I know, nginx can proxy HTTP and IMAP protocols. I'm not sure if it speaks SMTP...

**Reply by openaspace** [reply]
> As far as I know, nginx can proxy HTTP and IMAP protocols. I'm not sure if it speaks SMTP...

I will use: https://www.proxmox.com/en/proxmox-mail-gateway/get-started
